manifests/charts/istio-cni/values.yaml

```diff
@@ -23,6 +23,13 @@ cni:
   # Custom annotations on pod level, if you need them
   podAnnotations: {}
 
+  # Kubernetes >=v1.11.0 will create two PriorityClass, including system-cluster-critical and
+  # system-node-critical, it is better to configure this in order to make sure your Istio pods
+  # will not be killed because of low priority class.
+  # Refer to https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+  # for more detail.
+  priorityClassName: system-cluster-critical
```

Comment:
`system-cluster-critical` =  `description: Used for system critical pods that must run in the cluster, but can be moved to another node if necessary.`

`system-node-critical` = `  description: Used for system critical pods that must not be moved from their current node.`

I think this should be `system-node-critical` because `istio-cni` can NOT be moved from its node.

Reply:
@howardjohn Any concerns with changing the default priorityClassName to `system-node-critical`. It is a better default given that `istio-cni` is an extremely critical piece of networking in the service mesh

Reply:
Is there actually a difference in Kubernetes implementation for daemonset? I don't see how "but can be moved to another node if necessary" works for a daemonset.

Reply:
Looks like other CNIs are `system-node-critical`, so seems reasonable